cohort-engine/src/test/java/com/ibm/cohort/engine/CqlEngineWrapperTest.java

```diff
@@ -733,6 +735,56 @@ public void testInvalidLibraryName() throws Exception {
 				});
 	}
 	
+	@Test
+	public void testCQLTranslationCustomIGWithTargetUrl()  throws Exception{
+		FhirServerConfig fhirConfig = getFhirServerConfig();
+
+		mockFhirResourceRetrieval("/metadata", getCapabilityStatement());
+
+		Patient patient = getPatient("123", Enumerations.AdministrativeGender.FEMALE, "1978-05-06");
+		patient.addExtension(new Extension("http://fakeIg.com/fake-extension", new StringType("fakeValue")));
+		mockFhirResourceRetrieval(patient);
+
+		Library root = getLibrary("test", "cql/ig-test/test.cql");
+		Library helpers = getLibrary("FHIRHelpers", "cql/includes/FHIRHelpers.cql", "text/cql",
+									 "cql/includes/FHIRHelpers.xml", "application/elm+json");
+
+		RelatedArtifact related = new RelatedArtifact();
+		related.setType(RelatedArtifactType.DEPENDSON);
+		related.setResource("/Library/" + helpers.getId());
+		root.addRelatedArtifact(related);
```

Comment:
I wouldn't block merge for this, but it would be a nice-to-have to pull this out into helper method(s) in BaseFhirTest as it something I can see being needed in multiple tests.